man/systemd.network.xml

```diff
@@ -228,6 +228,39 @@
           if <literal>RequiredForOnline=no</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ActivationPolicy=</varname></term>
+        <listitem>
+          <para>Specifies the policy for <command>systemd-networkd</command> managing the link
+          administrative state. Specifically, this controls how <command>systemd-networkd</command>
+          changes the network device's <literal>IFF_UP</literal> flag, which is sometimes
+          controlled by system administrators by running e.g., <command>ip set dev eth0 up</command>
+          or <command>ip set dev eth0 down</command>.</para>
+
+          <para>Takes one of <literal>start-up</literal>, <literal>always-up</literal>,
+          <literal>manual</literal>, <literal>always-down</literal>, <literal>start-down</literal>,
+          or <literal>bound</literal>. When <literal>manual</literal>, <command>systemd-networkd</command>
+          will not change the link's admin state at all; the system administrator must bring the
+          interface up or down manually, as desired. When <literal>start-up</literal> or
+          <literal>start-down</literal>, <command>systemd-networkd</command> will set the link
+          admin state up or down, respectively, when <command>systemd-networkd</command> starts
+          (or, when the interface is first detected, if that occurs after <command>systemd-networkd</command>
+          starts), but it will not change the flag after the first time. When <literal>always-up</literal>
+          or <literal>always-down</literal>, <command>systemd-networkd</command> will set the
+          admin state up or down, respectively, when <command>systemd-networkd</command> starts
+          as well as any time <command>systemd-networkd</command> detects a change in the
+          administrative state. When <varname>BindCarrier=</varname> is also set, this is
+          automatically set to <literal>bound</literal> and any other value is ignored.</para>
+
+          <para>The administrative state is not the same as the carrier state, so using
+          <literal>always-up</literal> does not mean the link will never lose carrier. The link
+          carrier depends on both the administrative state as well as the network device's physical
+          connection. However, to avoid reconfiguration failures, when using <literal>always-up</literal>,
+          <varname>IgnoreCarrierLoss=</varname> is forced to true.</para>
+
+          <para>The default is <literal>start-up</literal>.</para>
```

Comment:
I think this para should be moved before the previous one.

Reply:
ack, i moved it to "When <literal>up</literal> (the default)"